packages/wallet/test/services/TransactionReemiter.test.ts

```diff
@@ -110,21 +123,26 @@ describe('TransactionReemiter', () => {
       const submitting$ = cold<NewTxAlonzoWithSlot>('------|');
       const transactionReemiter = createTransactionReemitter({
         confirmed$,
+        logger,
         rollback$,
         store,
-        submitting$
+        submitting$,
+        tipSlot$
       });
-      expectObservable(transactionReemiter).toBe('--a--c|', { a: volatileA, c: volatileC });
+      expectObservable(transactionReemiter).toBe('--a---', { a: volatileA });
```

Comment:
Could you please add 1 more valid transaction to be emitted after `c`? Just want to make sure that observable is still running after the error.

Reply:
Great catch @mkazlauskas. Thanks!
There's a bug there, it does not continue running.
I'm working on the fix